### PR TITLE
Update download-sql-server-data-tools-ssdt.md

### PR DESCRIPTION
**Pull request by stuartpa**
Reporting Services extensions for VS2022 are now available

### CODE REVIEW

**Comment by PRMerger3** [comment]
@stuartpa : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Court72** [comment]
#label:"aq-pr-triaged"

File: docs/ssdt/download-sql-server-data-tools-ssdt.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ monikerRange: ">=aps-pdw-2016||=azuresqldb-current||=azure-sqldw-latest||>=sql-s
 
 ### Changes in SSDT for Visual Studio 2022
 
-The core SSDT functionality to create database projects has remained integral to Visual Studio.  The extensions for Analysis Services, Integration Services, and Reporting Services projects are not available for Visual Studio 2022 at this time.
```
Comment (dzsquared) [inline docs/ssdt/download-sql-server-data-tools-ssdt.md:27]: still have a stray comma
"The extensions for Analysis Services and Integration Services projects are not available for Visual Studio 2022 at this time."

**Review by dzsquared — CHANGES_REQUESTED** [review]

**Comment by PRMerger17** [comment]
@stuartpa : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by dzsquared** [comment]
#sign-off

**Review by dzsquared — APPROVED** [review]

**Pull request merged**